### PR TITLE
Make profile image url absolute

### PR DESCRIPTION
**Pull request by amirkarimi**
https://github.com/jaywink/socialhome/issues/327

### CODE REVIEW

File: socialhome/users/models.py [hunk 1]
```diff
@@ -272,15 +272,25 @@ def get_last_name(self):
         return ""
 
     @staticmethod
+    def absolute_remote_url(remote_profile, image_name):
```
Comment (jaywink) [inline socialhome/users/models.py]: Could the signature be `absolute_image_url(profile, image_name)`? Reason: while we're using this for remote profile parsing, the functionality in the method doesn't actually have anything strictly to do with remote profiles. But it does have everything to do with image url's, so calling it `absolute_image_url` would make it easier to see right from the name what it does. With the current name my first thought is that this returns "an absolute url for the remote profile itself".

Also, a short docstring would be nice, just to indicate what it does (= returns absolute version of image ulr of given size).

**Review by jaywink — CHANGES_REQUESTED** [review]
Thanks for working on this! One change request, then good to go!

**Review by jaywink — APPROVED** [review]

**Comment by jaywink** [comment]
Perfect! Thanks for this <3 Did you also want to work on removing the `safer_` method usages (requires also doing a data migration for any profile image urls that are relative) or shall I log it as a follow-up issue?

**Comment by amirkarimi** [comment]
@jaywink I would like to work on it but as I might be a little bit busy, I think it's better to create another issue. I'll pick it up ASAP.

**Comment by jaywink** [comment]
@amirkarimi logged #374 , sorry forgot about this for a while. If you're interested in tackling it at some point. Thanks again for your contribution!

**Pull request merged**